### PR TITLE
Rst 419 421 court locations

### PR DESCRIPTION
**Pull request by MaxAwan**

### CODE REVIEW

File: courtfinder/courts/templates/courts/court.jinja [hunk 1]
```diff
@@ -88,13 +88,21 @@
...
+          {% if court.cci_code %}
+          <dt>County Court location code</dt>
+          <dd>{{ court.cci_code }}</dd>
+          {% endif %}
+          {% if court.magistrate_code %}
+          <dt>Magistrates' Court location code</dt>
+          <dd>{{ court.magistrate_code }}</dd>
+          {% endif %}
           {% if court.dx_number %}
           <dt><abbr title="document exchange">DX</abbr>:</dt>
```
Comment (richerry) [inline courtfinder/courts/templates/courts/court.jinja]: We should be consistent with the existence of titles.

File: courtfinder/courts/views.py [hunk 2]
```diff
@@ -153,7 +154,7 @@ def leaflet (request, slug, leaflet_type):
 def list_format_courts(courts):
     return [{'name':court.name,
              'slug':court.slug,
-             'numbers': ', '.join(filter(None,('#'+str(court.number) if court.number else None, 'CCI: '+str(court.cci_code) if court.cci_code else None)))
+             'numbers': ', '.join(filter(None,('Crown #'+str(court.number) if court.number else None, 'County #'+str(court.cci_code) if court.cci_code else None, "Magistrates #"+str(court.magistrate_code) if court.magistrate_code else None)))
```
Comment (richerry) [inline courtfinder/courts/views.py]: PEP8 please!

**Comment by robyoung** [comment]
Could you provide a better description of the change please?

File: courtfinder/search/ingest.py [hunk 3]
```diff
@@ -49,6 +49,7 @@ def courts(cls, courts, database_name="default"):
             court = Court(
                 admin_id=court_obj['admin_id'],
                 cci_code=court_obj.get('cci_code', None),
+                magistrate_code=court_obj.get('magistrate_code', None),
```
Comment (robyoung) [inline courtfinder/search/ingest.py]: Does this make the chaneg backwards compatibile and able to be deployed independently of the admin side change?

Comment (kgrecki) [inline courtfinder/search/ingest.py]: yes, I checked

File: courtfinder/courts/templates/courts/court.jinja (hunk 1 above)
Comment (kgrecki) [inline courtfinder/courts/templates/courts/court.jinja]: what's the inconsistency?

**Pull request closed without merging**